python/packages/autogen-core/src/autogen_core/components/models/_openai_client.py

```diff
@@ -365,6 +366,15 @@ async def create(
         create_args = self._create_args.copy()
         create_args.update(extra_create_args)
 
+        # Check to use beta client
+        use_beta_client = False
+
+        # Check if create_args contains response_format and if it is a Pydantic model
+        if "response_format" in create_args:
+            response_format = create_args["response_format"]
```

Comment:
The type of `response_format` needs to be fully specified in this function, and `response_format` needs to be passed in to the `parse` method to get the return value specified too (instead of ParsedChatCompletion[Any])

Reply:
done.